### ISSUE TITLE
the question of accuracy

### ISSUE THREAD

**Issue by TengFei75882341** [zZyan/race_gender_recognition#3]
Hi,
My loss is always greater than one, and the gender Accuracy is about 0.5, the race Accuracy is about 0.3,
my learning_rate is 0.0001
how I can improve it, which parameters should be change,
thanks a lot !!!
